examples/apps/generic_client/eth_generic_client.py

```diff
@@ -489,9 +491,14 @@ def Main(args=None):
 
     # Create work order
     verification_key = worker_obj.verification_key
-    wo_params = _create_work_order_params(worker_id, workload_id,
-                                          in_data, worker_obj.encryption_key,
-                                          session_key, session_iv)
+    code, wo_params = _create_work_order_params(
+                            worker_id, workload_id,
+                            in_data, worker_obj.encryption_key,
+                            session_key, session_iv)
+    if not code:
+        logger.error("Work order submission failed")
+        logger.error(wo_params)
```

Comment:
log wo_params in the previous log only.

Reply:
thanks Manju, Second log is not required. It will be removed.